assets.go

```diff
@@ -2,4 +2,8 @@
 
 package main
 
+import "net/http"
+
 const production = false
```

Comment:
I could've called it `dev` instead and inverted the logic, but I decided to explicitly call it `production` instead.

The way I see it, being in "development mode" turns off "production mode". However, there may be other things that also turn off "production mode". I want "production mode" to be on by default, unless I explicitly turn it off. Which is what's happening here.

Primarily, it's because when I'm doing production-y things, I want the logic to be like `if production { doProductionStuff(); }`, not `if !dev { doProductionStuff(); }`. I suppose I could write a func `isProductionMode() bool { return !dev }` but that just doesn't seem like an improvement over what I have. What do you think?